Create troubleshooting section about ephemeral storage use (#1054)
astronomer/docs: astro/test-and-troubleshoot-locally.md

- Misconfigured Dockerfile or Docker override file.
- Misconfigured Airflow settings.

> ### Astro/ Airflow runs out of local storage for files

Reviewer: ```suggestion
### Local file storage full on Astro or Airflow
```

> By design, workers on the Astro data plane include very limited ephemeral storage for Airflow tasks. Both Astronomer and the [Apache Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#communication) do not recommend creating or storing files in Airflow's local file system, as there's no guarantee that the storage will be sufficient for your tasks and persist between worker Pods.

Reviewer: ```suggestion
Workers on the Astro data plane include very limited ephemeral storage for Airflow tasks. Both Astronomer and the [Apache Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#communication) do not recommend creating or storing files in the Airflow local file system, as it's unlikely that there will be sufficient storage for your tasks and and that it will persist between worker Pods.
```

> To create or store files from Airflow, Astronomer recommends creating an Airflow connection to an external storage service such as Amazon S3 and using it in place of local file storage. For more guidance on storing data in Airflow, see Astronomer's guide on [passing data between Airflow tasks](https://www.astronomer.io/guides/airflow-passing-data-between-tasks/).

Reviewer: ```suggestion
To create or store files from Airflow, Astronomer recommends creating an Airflow connection to an external storage service such as Amazon S3. For more guidance on storing data in Airflow, see  [Passing Data Between Airflow Tasks](https://www.astronomer.io/guides/airflow-passing-data-between-tasks/).
```

> If you experience trouble implementing an alternative storage system for Airflow, contact [Astronomer support](https://support.astronomer.io/) and describe your use case.

Reviewer: ```suggestion
If you experience issues implementing an alternative storage system for Airflow files, contact [Astronomer support](https://support.astronomer.io/).
```

Reviewer 2: I would keep it as just Airflow rather than Airflow files. Airflow files makes it sound like it is Airflow system files rather than any user-created/downloaded files.
